elastic.js/index.d.ts

```diff
@@ -1,9 +1,11 @@
 // Type definitions for elastic.js v1.2.0
 // Project: https://www.npmjs.com/package/elastic.js
-// Definitions by: Oleksii Trekhleb <https://ua.linkedin.com/in/trekhleb>
+// Definitions by: Oleksii Trekhleb <https://ua.linkedin.com/in/trekhleb>, Blake Smith <https://github.com/bfsmith>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare module elasticjs {
+export = elasticjs;
```

Comment:
you also need `export as namespace elasticjs`. this allows the use of this as a global, e.g. in a website. 

The docs suggest that this is a valid scenario:https://www.elastic.co/blog/client-for-node-js-and-the-browser